bower.json

```diff
@@ -14,6 +14,7 @@
     "dist/angular-leaflet-directive.js"
   ],
   "dependencies": {
+    "Leaflet.vector-markers": "https://github.com/hiasinho/Leaflet.vector-markers",
```

Comment:
I could not find it in the Bower repository, so I used the GitHub URL instead.

I'll add an SHA.